tests/test_authorization_header.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from tests.test_case import TestCase
```

Comment:
You can replace this with:
```py
from unittest import TestCase
```

That way, database and api initialisations will be skipped, which is not needed for this method. This makes tests rocking fast! ⚡️